README.md

```diff
@@ -320,43 +362,32 @@ For more information on limitations and recommendations, please [read](https://d
    1. Connect to MySQL database using master logins and create a new user with the following<br>
    `CREATE USER example_user_name IDENTIFIED WITH AWSAuthenticationPlugin AS 'RDS';`
 
-| Parameter       | Value           | Required      | Description  |
-| ------------- |:-------------:|:-------------:| ----- |
-|`useAwsIam` | Boolean | No | Set to true if you would like to use AWS IAM database authentication<br/><br/>**Default value:** `false` |
+| Parameter       | Value           | Required      | Description  | Default Value |
+| ------------- |:-------------:|:-------------:|:-------------:| --------- |
+|`useAwsIam` | Boolean | No | Set to true to enable AWS IAM database authentication | `false` |
 
 ###### Sample Code
 ```java
-import java.sql.Connection;
-import java.sql.DriverManager;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Statement;
+import java.sql.*;
 import java.util.Properties;
 
-import software.aws.rds.jdbc.mysql.shading.com.mysql.cj.conf.PropertyKey;
-import software.aws.rds.jdbc.mysql.Driver;
-
 public class AwsIamAuthenticationSample {
 
-   private static final String CONNECTION_STRING = "jdbc:mysql:aws://database-mysql-name.cluster-XYZ.us-east-2.rds.amazonaws.com";
-   private static final String USER = "example_user_name";
+   private static final String CONNECTION_STRING = "jdbc:mysql:aws://database-mysql-name.cluster-XYZ.us-east-2.rds.amazonaws.com:3306/employees";
```

Comment:
Do the databases have to be in us-east-2? @matthewh-BQ I think we should be more generic.